experiments/JSDNS.jsm.js

```diff
@@ -60,17 +60,12 @@
 /* eslint complexity: "off" */
 /* eslint no-magic-numbers: "off" */
 /* eslint mozilla/mark-exported-symbols-as-used: "error" */
-
+/* global Services */
 
 var EXPORTED_SYMBOLS = [
 	"JSDNS"
 ];
 
-
-// @ts-expect-error
-var { Services } = ChromeUtils.import("resource://gre/modules/Services.jsm");
```

Comment:
In TB 91 the import of `Services.jsm` seems to be still needed here, as this is a separate jsm module and not in the context of the WebExtensions experiments.

Reply:
Thanks!

Yeah, JSM global doesn't have `Services` global variable until 104 (https://bugzilla.mozilla.org/show_bug.cgi?id=1667455)
I'll fix the code and also the commit message.